### PR TITLE
Integrate prometheus. Allow custom metrics

### PR DESCRIPTION
**Pull request by allevo**
#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [x] the documentation is updated or intregrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: docs/main-entrypoint.md [hunk 1]
```diff
@@ -73,6 +73,30 @@ Both of these entpoint conform to the JSON schema that you can find [here][statu
 Both of this endpoints are set to permanently run on log level `silent` for decreasing the amount of noise in the
 logs during the deployment.
 
+## Prometheus Metrics
+
+By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there're some process, garbage collection and http information.
```
Comment (davidebianchi) [inline docs/main-entrypoint.md:78]:
```suggestion
By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there are some process, garbage collection and http information.
```

Comment (fredmaggiowski) [inline docs/main-entrypoint.md]: What do you mean by "In this metrics there're some process, garbage collection and http information." ? Is it the purpose of the endpoint? Can we write it more clearly?

Something on the line of:

> The endpoint is responsible of exposing relevant information about running processes, http request information (current load, errors, etc)

What do you think?

Comment (fredmaggiowski) [reply docs/main-entrypoint.md]: Is it only an empty route or does it have some information provided by default? Can we clarify this?

Also is there some doc about the shape of the object that must be returned by the endpoint?

Comment (allevo) [reply docs/main-entrypoint.md]: `The response body contains process, garbage collection and http information.`
?

File: docs/main-entrypoint.md [hunk 2]
```diff
@@ -73,6 +73,30 @@ Both of these entpoint conform to the JSON schema that you can find [here][statu
 Both of this endpoints are set to permanently run on log level `silent` for decreasing the amount of noise in the
 logs during the deployment.
 
+## Prometheus Metrics
+
+By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there are some process, garbage collection and http information.
```
Comment (allevo) [inline docs/main-entrypoint.md:78]:
```suggestion
By default `lc39` exposes `/-/metrics` endpoint for [Prometheus](https://prometheus.io/). The response body contains process, garbage collection and http information
```

**Pull request merged**